editor/src/components/editor/store/editor-update.tsx

```diff
@@ -116,10 +116,8 @@ export function runSimpleLocalEditorAction(
     case 'CLOSE_POPUP':
       return UPDATE_FNS.CLOSE_POPUP(action, state)
     case 'PASTE_JSX_ELEMENTS':
-      return UPDATE_FNS.PASTE_JSX_ELEMENTS(action, state, dispatch)
+      return UPDATE_FNS.PASTE_JSX_ELEMENTS(action, state, dispatch, builtInDependencies)
     case 'COPY_SELECTION_TO_CLIPBOARD':
-      // side effect 😟
-      setClipboardData(createClipboardDataFromSelection(state))
```

Comment:
is this moved somewhere else? it's not clear for me from the PR description

Reply:
That's what this relates to:
"Shifted the side effecting `setClipboardData` inside the action parameter of `toastOnUncopyableElementsSelected` so that we don't run the side effect regardless of if it's allowed or not."